main.rmd

```diff
@@ -150,6 +165,7 @@ mapply(compare.to.index(xs.returns$CROBEX), xs.returns[investment_funds])
 
 ## Testovi povrata mirovinskih fondova u odnosu na CROBEX
 
+Za H0 hipotezu uzimamo kako su sredine mirovinskih fondova i CROBEX indeksa jednake. Iz sljedećih testova zaključujemo da možemo odbaciti H0 hipotezu uz nivo značajnosti 10%.k
```

Comment:
10%.k

možda dodat nešto da uz uobičajenu p-vrijednost od 5% samo se jedan fond po srednjim prinosima zadnjih 7 godina značajno razlikuje od CROBEX-a.